plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Tasks.js

```diff
@@ -253,9 +258,7 @@ Ext.define('NX.coreui.controller.Tasks', {
         list = me.getList();
 
     if (list) {
-      me.getTaskTypeStore().load(function() {
-        me.reselect();
-      });
+      me.getStore('TaskType').load();
```

Comment:
Current approach is wonky for sure. Just spent a few minutes playing with autoSync, but couldn’t get it to work. It’s probably not worthwhile enough to hold up this PR, but let me know if you figure it out.